app/main.py

```diff
@@ -1,3 +1,65 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.customer import Customer
+from app.car import Car
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json") as data_in:
+        data_from_file = json.load(data_in)
+        fuel_coast = Car(fuel_price=data_from_file["FUEL_PRICE"])
+        customers = [
+            Customer(customer) for customer in data_from_file["customers"]
+        ]
+        shops = [Shop(shop) for shop in data_from_file["shops"]]
+
+        for customer in customers:
+            customer_home = customer.location.copy()
+            print(f"{customer.name} has {customer.money} dollars")
+            dict_shop_and_coast = {}
+            for shop in shops:
+                coast_to_shop = customer.coast_of_trip_to_shop(
+                    shop,
+                    fuel_coast
+                )
+                dict_shop_and_coast[coast_to_shop] = shop
+                print(f"{customer.name}'s trip to "
+                      f"the {shop.name} costs {coast_to_shop}")
```

Comment:
Like for example, you can move this logic to another method or function in customer module